components/scream/src/share/atm_process/atmosphere_process_group.cpp

```diff
@@ -144,25 +144,162 @@ void AtmosphereProcessGroup::finalize_impl (/* what inputs? */) {
 }
 
 void AtmosphereProcessGroup::
-set_required_group_impl (const FieldGroup<const Real>& group)
-{
+set_required_field (const Field<const Real>& f) {
+  // Find the first process that requires this group
```

Comment:
This assumes sequential splitting right?  Will this be an issue when we develop parallel splitting in the model?

Reply:
Added a check.